libknet/tests/api_knet_link_get_config.c

```diff
@@ -107,7 +107,7 @@ static void test(void)
 	memset(&get_src, 0, sizeof(struct sockaddr_storage));
 	memset(&get_dst, 0, sizeof(struct sockaddr_storage));
 
```

Comment:
we need to a add a test here where we pass NULL in transport to verify that the API is returning the correct error.